tools/data_gen/src/main.rs

```diff
@@ -0,0 +1,173 @@
+#[macro_use]
+extern crate lazy_static;
+
+use argh::FromArgs;
+use calyx::{errors::CalyxResult, frontend, ir};
+use serde::ser::{Serialize, Serializer};
+use serde_json::{json, Map, Value};
+use std::borrow::Borrow;
+use std::cell::{Ref, RefCell};
+use std::collections::HashMap;
+use std::path::{Path, PathBuf};
+use std::rc::Rc;
+
+lazy_static! {
+    static ref SIZEMAP: HashMap<&'static str, Vec<&'static str>> = {
+        let mut m = HashMap::new();
+        m.insert("std_mem_d1", vec!["SIZE"]);
```

Comment:
Instead of the insert calls, you can define this as a `vec![("std_mem_d1", "SIZE"), ...]` and call `.into_iter` followed by `.collect::<HashMap<_, _>>()` to transform it into a hash map.